src/write_gcm.c

```diff
@@ -2813,7 +2824,8 @@ static void wg_json_RFC3339Timestamp(json_ctx_t *jc, cdtime_t time_stamp);
 //   string collectd_version = 3;
 //   repeated CollectdPayload collectd_payloads = 4;
 // }
-static int wg_json_CreateCollectdTimeseriesRequest(_Bool pretty,
+static int wg_json_CreateCollectdTimeseriesRequest(
```

Comment:
Are you planning to add partial success support to the other path (the one that uses `CreateTimeSeries`) in a later PR?